config/default.yml

```diff
@@ -1043,13 +1043,6 @@ Style/WordArray:
   # The regular expression WordRegex decides what is considered a word.
   WordRegex: !ruby/regexp '/\A[\p{Word}\n\t]+\z/'
 
-# RequestReferer enforces consistent request.referer usage.
```

Comment:
Note: I removed this comment, since this file has no comment for any of the other Rails cops. Please let me know if I should add it back.